### PR TITLE
Zap up down

### LINKED ISSUE #9: zap to next/previous channel

### PR DESCRIPTION
**Pull request by fabsi88**
close #9

### CODE REVIEW

File: src/d/alexa-openwebif/dub.sdl [hunk 1]
```diff
@@ -6,5 +6,6 @@ license "MIT"
 dependency "vibe-d" version="~>0.7.30"
 dependency "alexa-skill-kit-d" version="~>0.2"
 //dependency "alexa-skill-kit-d" version="~>master" path="../../../../alexa-skill-kit-d"
-dependency "openwebif-client-d" version="~>0.1"
+dependency "openwebif-client-d" version="~>0.1.1"
```
Comment (Extrawurst) [inline src/d/alexa-openwebif/dub.sdl]: why specific version ? please stick to 0.1 without specific patch version

File: src/d/alexa-openwebif/source/main.d [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import std.stdio;
 import std.xml;
 import std.string;
+import std.algorithm.searching;
```
Comment (Extrawurst) [inline src/d/alexa-openwebif/source/main.d]: please use local import when u need this module just in your new method and use selective importing of just the symbol used `import std.algorithm.searching:countUntil;`

File: src/d/alexa-openwebif/source/main.d [hunk 3]
```diff
@@ -243,49 +244,98 @@ final class OpenWebifSkill : AlexaSkill!OpenWebifSkill
...
+    auto targetChannel = event.request.intent.slots["targetChannel"].value;
+    Subservice matchedServices; 
+    auto switchedTo = "nichts";
+   
+    if(targetChannel.length > 0)
+    {
+      auto allservices = apiClient.getallservices();
+      immutable auto maxIndex = allservices.services[0].subservices.length;
+
+      if (targetChannel == "up" || targetChannel == "down")
```
Comment (Extrawurst) [inline src/d/alexa-openwebif/source/main.d]: please split up this too long method at least into the up/down and the "normal" zapTo part

File: src/d/alexa-openwebif/dub.sdl (hunk 1 above)
Comment (fabsi88) [inline src/d/alexa-openwebif/dub.sdl]: fixed

File: src/d/alexa-openwebif/source/main.d (hunk 2 above)
Comment (fabsi88) [inline src/d/alexa-openwebif/source/main.d]: fixed

File: src/d/alexa-openwebif/source/main.d (hunk 3 above)
Comment (fabsi88) [inline src/d/alexa-openwebif/source/main.d]: fixed

**Pull request merged**